hieradata/common.yaml

```diff
@@ -2,6 +2,7 @@
 HIERA_SAFETY_CHECK: true
 
 app_domain: 'dev.gov.uk'
+app_domain_internal: 'dev.govuk-internal.digital'
```

Comment:
`digital` doesn't have any meaning on non-AWS, maybe we should set app_domain_internal to `dev.gov.uk`?

Reply:
Sure.